@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/mpd/fn_mpdLFUELDraw.sqf

```diff
@@ -8,30 +8,30 @@ params["_heli"];
 #define KGTOLBS 2.20462
 
 
-private _forwardCellWeight = [_heli] call fza_fnc_sfmplusSetFuel select 0;
-private _aftCellWeight     = [_heli] call fza_fnc_sfmplusSetFuel select 1;
+private _forwardCellWeight = [_heli] call fza_sfmplus_fnc_fuelSet select 0;
+private _aftCellWeight     = [_heli] call fza_sfmplus_fnc_fuelSet select 1;
 private _totFuelCellWeight = _forwardCellWeight + _aftCellWeight;
 _forwardCellWeight = _forwardCellWeight * KGTOLBS;
 _aftCellWeight     = _aftCellWeight * KGTOLBS;
 _totFuelCellWeight = _totFuelCellWeight * KGTOLBS;
 
 private _fuelWeight = _totFuelCellWeight;
 
-private _FFVal = [_heli] call fza_fnc_sfmplusGetData select 2;
-_FFVal = _FFVal * 7936.64;
+private _eng1FF = _heli getVariable "fza_sfmplus_engFF" select 0;
+private _eng2FF = _heli getVariable "fza_sfmplus_engFF" select 1;
 
 private _eng1FuelCons = 0;
-private _eng1State    = ((_heli getVariable "fza_ah64_engineStates") select 0) select 0;
-if (isEngineOn _heli && (_eng1State != "OFF")) then {
-	_eng1FuelCons = _FFVal / 2;
+private _eng1State    = _heli getVariable "fza_sfmplus_engState" select 0;
+if (isEngineOn _heli && (_eng1State != "OFF" || _eng1State != "DEST")) then {
+	_eng1FuelCons = _eng1FF * 7936.64;
```

Comment:
Can you make this a constant, so a the top of the file write

```sqf
#define FUEL_FLOW_LBS_PER_HR 7936.64
```

and then when you want to use it you can just do

```sqf
_eng1FuelCons = _eng1FF * 7936.64
```